src/spec/Paths.php

```diff
@@ -183,6 +183,7 @@ public function getErrors(): array
      * @return boolean true on success or false on failure.
      * The return value will be casted to boolean if non-boolean was returned.
      */
+    #[\ReturnTypeWillChange]
```

Comment:
given that this library is not compatible with PHP 5, as far as I see it would be better to make return types compatible instead of adding the annotation which only hides the deprecation warning but does not actually fix the problem.

see https://php.watch/versions/8.1/internal-method-return-types

Reply:
just seen that this is what https://github.com/cebe/php-openapi/pull/143 does, will check that in more detail later.

Reply:
I intended this as a middle step in order to keep BC.

After this there's https://github.com/cebe/php-openapi/pull/143 to be merged